index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './lib/config';
+export type { Config } from './lib/config';
+export type { Message } from './lib/message';
 export * from './lib/json_object';
```

Comment:
This file is also just defining and exporting types/interfaces:
https://github.com/mycase/chulie/blob/52bb08162ddaf7a494fac1b44cd528d2844c330e/lib/json_object.d.ts#L1-L7

so do we have to declare this one as a type export, too?

Reply:
this would have the same issue.  However, it looks like that we don't need to expose `JsonObject` to end user.  Removed in dd2a85b